cmd/calyptia/top_project.go

```diff
@@ -128,8 +128,8 @@ func (m ProjectModel) loadData(ctx context.Context, skipError bool) tea.Cmd {
 				for _, agent := range agents {
 					agent := agent
 					// skip metrics request if agent is offline
-					if agent.FirstMetricsAddedAt.IsZero() || agent.FirstMetricsAddedAt.After(metricsStart) ||
-						agent.LastMetricsAddedAt.IsZero() || agent.LastMetricsAddedAt.Before(metricsStart) {
+					if agent.FirstMetricsAddedAt == nil || agent.FirstMetricsAddedAt.IsZero() || agent.FirstMetricsAddedAt.After(metricsStart) ||
+						agent.LastMetricsAddedAt == nil || agent.LastMetricsAddedAt.IsZero() || agent.LastMetricsAddedAt.Before(metricsStart) {
```

Comment:
This sentence is growing in new conditions (`||`) impairing readability maybe we can simplify it or divide for readability sake

Reply:
https://github.com/calyptia/cli/issues/297